src/backend/brokers/go.emails/inbound.go

```diff
@@ -151,6 +152,19 @@ func (b *EmailBroker) processInbound(in *SmtpEmail, raw_only bool) {
 				if b.Config.LogReceivedMails {
 					log.Infof("EmailBroker : NATS inbound request successfully handled for user %s : %s", rcptId.String(), (*nats_ack)["message"])
 				}
+
+				notif := Notification{
+					Emitter:   "lmtp",
```

Comment:
I'm not agree using lmtp as emitter value. Should be better to use a more generic value instead `message`.

Reply:
We discussed this wording with @iamdey and thougth that "message" is too much generic ;-)  Email, DM, SMS whatever else, are "message" but they may be handled differently by the front end depending of from which connectors they are coming from : smtp, xmpp, etc.  
Giving a more specific indication of the emitter is helpful for the frontend to handle the message accordingly.

Reply:
OK let's go for lmtp (smtp is maybe better than an internal protocol, no ?)

Reply:
ok commit #f21d7f5